.github/workflows/notebook-deploy.yml

```diff
@@ -42,3 +42,6 @@ jobs:
 
           # Optional local file path to the repository
           repository: .
+
+          # Optional options appended to `git-push`
+          push_options: '--force'
```

Comment:
How come this is needed?

Reply:
It was in the original example of the specific Action's Readme (optional, of course), but I didn't include before. Probably it's not needed, this just ensures that the `gh-pages` branch is correct, I believe. Basically I've run some scenarios theoretically, and this seemed like little downside & no effect on other parts of code, so added it. Can remove it again, until we find that  we run into any issues.

Reply:
Cool no worries, was just curious.